build.sbt

```diff
@@ -71,3 +71,5 @@ lazy val uring = crossProject(NativePlatform, JVMPlatform)
     ),
     fork := true
   )
+  .jvmConfigure(_.in(file("uring/jvm")))
+  .nativeConfigure(_.in(file("uring/native")))
```

Comment:
Or this. What is the problem?

Reply:
I thought we need to indicate in sbt jvm/native/shared for the tests but maybe we don't 😅